utils/vlog/tests/muxes/golden.pb_type.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version='1.0' encoding='utf-8'?>
+<pb_type xmlns:xi="http://www.w3.org/2001/XInclude" name="use_mux" num_pb="1">
+  <input name="a" num_pins="1"/>
+  <input name="b" num_pins="1"/>
+  <input name="c" num_pins="1"/>
+  <output name="o1" num_pins="1"/>
+  <output name="o2" num_pins="1"/>
+  <interconnect>
+    <mux name="mux1">
+      <port name="a" type="input">
+        <metadata>
+          <meta key="fasm_mux">I0</meta>
+        </metadata>
+      </port>
+      <port name="b" type="input">
+        <metadata>
+          <meta key="fasm_mux">I1</meta>
+        </metadata>
+      </port>
+      <port name="o1" type="output"/>
+      <metadata>
+        <meta key="type">bel</meta>
+        <meta key="subtype">routing</meta>
+      </metadata>
+    </mux>
+    <mux name="mux2">
```

Comment:
The xmlsort.xsl script transforms the output into;
```xml
    <mux name="mux2" input="use_mux.b use_mux.c " output="use_mux.o2 ">
      <metadata>
        <meta key="fasm_mux">
            use_mux.b : I0
            use_mux.c : I1
        </meta>
        <meta key="type">bel</meta>
        <meta key="subtype">routing</meta>
      </metadata>
    </mux>
```

Reply:
Diff test please

Reply:
This was extracted into https://github.com/SymbiFlow/symbiflow-arch-defs/pull/728